### PR TITLE
integration tests for services

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes KILTProtocol/ticket#617
Adds automated calls to endpoints as integration testing

## How to test:
run tests with `yarn test:e2e`

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: src/config/config.service.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import * as dotenv from 'dotenv'
 import * as fs from 'fs'
-import { Injectable } from '@nestjs/common'
```
Comment (rflechtner) [inline src/config/config.service.ts:3]: unused import

File: src/contacts/contacts.controller.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export class ContactsController {
         throw new BadRequestException('bad signature for hash')
       }
     }
-    this.contactService.add(contact)
```
Comment (rflechtner) [inline src/contacts/contacts.controller.ts:45]: related to incorrect interfaces

**Review by LeonFLK — APPROVED** [review]
Beautiful

LGTM

File: src/contacts/contacts.controller.ts (hunk 2 above)
Comment (rflechtner) [reply src/contacts/contacts.controller.ts:45]: we'd be responding before the request has been fully processed if we don't await. Could silently fail and the requesting party would never know

**Pull request merged**